src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/BrokeredMessageToStringConverter.cs

```diff
@@ -2,54 +2,76 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
+using System.Globalization;
 using System.IO;
+using System.Runtime.Serialization;
+using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Xml;
 using Microsoft.Azure.WebJobs.Host.Converters;
 using Microsoft.ServiceBus.Messaging;
 
 namespace Microsoft.Azure.WebJobs.ServiceBus.Triggers
 {
     internal class BrokeredMessageToStringConverter : IAsyncConverter<BrokeredMessage, string>
     {
-        public Task<string> ConvertAsync(BrokeredMessage input, CancellationToken cancellationToken)
+        public async Task<string> ConvertAsync(BrokeredMessage input, CancellationToken cancellationToken)
         {
             if (input == null)
             {
                 throw new ArgumentNullException("input");
             }
+            Stream stream = input.GetBody<Stream>();
+            if (stream == null)
+            {
+                return null;
+            }
 
-            if (input.ContentType == ContentTypes.TextPlain ||
```

Comment:
You're removing all of these in this converter, but leaving them in the byte[] converter. Can we be consistent across all?